Fixed some small errors in CSS for modules and some errors in emulators in general.

favorit5503 opened the pull request:
Fixed some errors in emulators for the GM module - With Legion and BFA the GM module wont error out now.
Fixed some errors in emulator itself - Bad expansion naming
Fixed some CSS errors for the modules. - Wrong button sizes, etc.

mdjong1 commented on application/emulators/Trinity_rbac_wod_soap.php:
```diff
@@ -27,8 +27,7 @@ class Trinity_rbac_wod_soap implements Emulator
 	protected $expansions = array(
-		8 => "Battle for Azeroth",
-		7 => "Legion Allied Races",
+		7 => "Battle for Azeroth",
```
This kind of change doesn't belong in a CSS error fix PR. Make another PR for removing Legion Allied Races.

mdjong1 commented on application/emulators/trinity_rbac_bfa_soap.php:
```diff
@@ -50,7 +50,7 @@ class Trinity_rbac_bfa_soap implements Emulator
 		"guild_member" => "guild_member",
 		"guild" => "guild",
-		"gm_tickets" => "gm_ticket"
+		"gm_tickets" => "gm_bug"
```
Is this related to the CSS fix?

favorit5503 commented:
So you didnt see what i wrote to this PR. I made these 3 changes:
Fixed some CSS errors for the modules.
Fixed some errors in emulators for the GM module.
Fixed some errors in emulator itself.

mdjong1 commented:
I still think CSS changes and removing a whole expansion are different PR's

alexandre433 commented:
I agree, but Legion Allied Races, isn't a expasion, should just be Legion

favorit5503 commented:
> I agree, but Legion Allied Races, isn't a expasion, should just be Legion

Yes and this is why i am fixing this

mdjong1 commented:
> I agree, but Legion Allied Races, isn't a expasion, should just be Legion

Should still be a separate PR regardless :p

But has been noted that it will be separated in the future
![image](https://user-images.githubusercontent.com/4410453/80965281-8d55fe00-8e12-11ea-96ab-e29c1d30d272.png)
